aiofreepybox/api/home.py

```diff
@@ -0,0 +1,266 @@
+# Home structure : adapter > node > endpoint
+class Home:
+
+    def __init__(self, access):
+        self._access = access
+
+    home_endpoint_value_schema = {
```

Comment:
Having schemas to define the expected payload is an excellent idea, but I don't understand how they are defined: sometimes, the key is associated with a type, and sometimes with a value. For instance, in `create_home_node_rule_payload_schema`, `id` is associated with the **type** `int`, but `iconUrl` is associated with the **value** "empty string". I would expect the schemas to define fields with types (like `int`, which is `<class 'int'>`, or `str`), string representing types (like `'int'` or `'str'`) or maybe default values (like `0` or `''`).

Reply:
After looking at your other PRs, I think that having default values in the schemas may be better than having types.

Reply:
I agree it can be confusing, i'll set all `int` default values to 0